tests/cases/fourslash/jsDocPropertyDescription10.ts

```diff
@@ -0,0 +1,13 @@
+///<reference path="fourslash.ts" />
+
+//// class MultipleClass {
+////     /** Something generic */
+////     [key: number | symbol | `data-${string}` | `data-${number}`]: string;
+////     [key: string]: string;
+//// }
+//// function multipleClass(e: typeof MultipleClass) {
+////     console.log(e./*multipleClass*/anything);
+////     console.log(e./*multipleClass1*/something);
+//// }
+
+verify.quickInfoAt("multipleClass", "any");

```

Comment:
What is this test testing?

Reply:
That's a left over from a test that I should've removed